<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>816</width>
     <height>239</height>
    </rect>
   </property>
   <property name="title">
    <string>multiple choice questions(Non Exclusive)</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'PMingLiU'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;which animal should not stay at house&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="AcheckBox">
      <property name="text">
       <string>tiger</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="BcheckBox">
      <property name="text">
       <string>lion</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="CcheckBox">
      <property name="text">
       <string>cat</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="DcheckBox">
      <property name="text">
       <string>dog</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>257</y>
     <width>816</width>
     <height>239</height>
    </rect>
   </property>
   <property name="title">
    <string>multiple choice(Exclusive)</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'PMingLiU'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;which animal is not feline&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ACheckBox">
      <property name="text">
       <string>cat</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="BCheckBox">
      <property name="text">
       <string>dog</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="CCheckBox">
      <property name="text">
       <string>fish</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="DCheckBox">
      <property name="text">
       <string>pig</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>503</y>
     <width>816</width>
     <height>187</height>
    </rect>
   </property>
   <property name="title">
    <string>Choose the right answer</string>
   </property>
   <widget class="QRadioButton" name="Truebutton">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>130</y>
      <width>54</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>True</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Falsebutton">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>156</y>
      <width>56</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>False</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_3">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>27</y>
      <width>792</width>
      <height>96</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'PMingLiU'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;every dog has a tail&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="UploadButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>690</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Upload answer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
